### DISCUSSION TITLE
How to manage subscription-based sites ?

### DISCUSSION THREAD

**Discussion by lhoupert** [mmagnus/Pocket-Plus-Calibre-Plugin#37] [Ideas]
Thank you for this great plugin! I was wondering if there was a way to handle subscription-based website?

I found some information about it on https://help.getpocket.com/article/1065-saving-from-subscription-based-sites and I managed to add articles from subscription-based website in my Android app but I cannot see them properly in the ebook. 

It is certainly due to the fact that they don't appear properly on https://app.getpocket.com/ . I was wondering if there was a solution to go around this problem?

**Comment by mmagnus** [comment]
Dear @lhoupert, it's possible but it's ugly and complicated. This plugin will not manage multiple logins etc. 

One super hacky solution is to use my pocketX [1]. pocketX is a version of this plugin here I made for myself to read a Polish journal Polityka (behind paywall). 

The hack is that when this plugin reads your Pocket, it will detect links to Polityka (in my case) and will create an RSS feed that will be published (in my case) in Dropbox.

Then you have to find a plugin (Polityka.recipe) of Calibre that can be used to login in to your page of choice (this is independent of this plugin here), and ask this external plugin to read created RSS feed and manage login to the web site.

It can work, but it's very hacky, and it assumes that there is already a plugin for Calibre to manage login to page of your choice.

So if you're super motivated, you can try to pull pocketX and configure it for your page. I will try to help. However, find first if Calibre can manage login to the page you want to get articles from, or make your own [2].

Unfortunately, this is how this is right now :(

<img width="800" alt="Screenshot 2021-02-07 at 21 03 37" src="https://user-images.githubusercontent.com/118740/107158080-feb97300-6987-11eb-844a-d92b121b5bdb.png">
So the individual plugin you can set up logins, and inside this plugin (recipe) you can define the RSS feed created with PocketX.

```
    feeds          = [
        ('polityka.pl', 'http://dl.dropbox.com/s/27t8uhx4nlslggr/feed.xml'),
    ]
```

[1] https://github.com/mmagnus/Pocket-Plus-Calibre-Plugin/tree/master/pocketX
[2] https://manual.calibre-ebook.com/news.html
